src/System.Linq/src/System/Linq/ToCollection.cs

```diff
@@ -177,7 +177,7 @@ public static List<TSource> ToList<TSource>(this IEnumerable<TSource> source)
             return d;
         }
 
-        public static HashSet<TSource> ToHashSet<TSource>(this IEnumerable<TSource> source) => source.ToHashSet(comparer: null);
+        public static HashSet<TSource> ToHashSet<TSource>(this IEnumerable<TSource> source) => source.ToHashSet(null);
```

Comment:
Incidentally, the style here is that we tend to use named parameters with values of literal `null`, `true` or `false` and literal numbers where it can be rather obscure just what is null or false etc. at first glance, compared to say `ToHashSet(comparer)` where its easier to guess what sort of thing `comparer` is. I have to say I've begun to like the style, speaking as someone who didn't like named parameters when C# added them.